README.md

```diff
@@ -31,30 +34,103 @@ terraform {
 }
 ```
 
-Additional documentation, including available resources and their arguments/attributes can be found on the [Terraform documentation website](https://terraform.io/docs/providers/time).
+## Documentation, questions and discussions
+Official documentation on how to use this provider can be found on the
+[Terraform Registry](https://registry.terraform.io/providers/hashicorp/tls/latest/docs).
+In case of specific questions or discussions, please use the
+HashiCorp [Terraform Providers Discuss forums](https://discuss.hashicorp.com/c/terraform-providers/31),
+in accordance with HashiCorp [Community Guidelines](https://www.hashicorp.com/community-guidelines).
 
-## Developing the Provider
+We also provide:
 
-If you wish to work on the provider, you'll first need [Go 1.16 or later](http://www.golang.org) installed on your machine. This project uses [Go Modules](https://blog.golang.org/using-go-modules) making it safe to work with it outside of your existing [GOPATH](http://golang.org/doc/code.html#GOPATH).
+* [Support](.github/SUPPORT.md) page for help when using the provider
+* [Contributing](.github/CONTRIBUTING.md) guidelines in case you want to help this project
 
-### Building the Provider
+## Compatibility
 
-From the top directory of the repository:
+Compatibility table between this provider, the [Terraform Plugin Protocol](https://www.terraform.io/plugin/how-terraform-works#terraform-plugin-protocol)
+version it implements, and Terraform:
 
-```console
-$ go build
-```
+| Time Provider |     Terraform Plugin Protocol      | Terraform |
+|:-------------:|:----------------------------------:|:---------:|
+|   `>= 7.x`    |                `5`                 | `>= 0.12` |
```

Comment:
Believe we need to prefix all the `Time Provider` versions with a zero, for instance `0.7.0`.